utilities/ipplan-import.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+""" NIPAP import script for ipplan data.
+
+This script uses the standard export files base.txt and ipaddr.txt from ipplan.
+
+base.txt format:
+ip\t description\t netmask\t comment
+
+ipaddr.txt format:
+ip\t user\t location\t description\t hostname\t phone\t comment
+"""
+import configparser
+import os
+# use local pynipap, useful if we are developing
+import sys
+sys.path.append('../pynipap')
+import pynipap
+import csv
+import ipaddress
+
+
+def add_ip_to_net(networks, host):
+    """ Add hosts from ipplan to networks object. """
+    for network in networks:
+        if host['ipaddr'] in network['network']:
+            network['hosts'].append(host)
+            return
+
+
+def get_networks(base_file, ipaddr_file):
+    """ Gather network and host information from ipplan export files. """
+    networks = []
+
+    base = open(base_file, 'r')
+
+    csv_reader = csv.reader(base, delimiter='\t')
+
+    buffer = ""
+    for row in csv_reader:
+
+        # Fixes quotation bug in ipplan exporter for networks
+        if len(networks) > 0 and len(buffer) > 0:
+            networks[-1]['comment'] += " ".join(buffer)
```

Comment:
No, the script handles multi-line comments when they are quoted correctly. However there is a bug in the export script in ipplan that does not quote multi-line comments correctly for the base.txt file.

Here's the bug: https://github.com/hetznerZA/ipplan/blob/master/admin/exportbase.php#L75